modules/installation-and-upgrade/pages/container-deployment/generate_proxy_config.adoc

```diff
@@ -0,0 +1,89 @@
+// Add section on how to generated the proxy configuration files
+
+[[proxy-setup-containers-generate-config]]
+== Generate the Proxy Configuration
+
+The configuration archive of the {productname} Proxy is generated by the {productname} Server.
+Each additional Proxy requires its own configuration archive.
+
+[NOTE]
+====
+2 GB represents the default proxy squid cache size. This will need to be adjusted for your environment.
```

Comment:
This note is without any context before, so it could be confusing for the user. Maybe, we should move it close to where we are describing squid cache size.

Reply:
This note in in the section about generating the config, and should apply to all available methods

Reply:
In this case, we'd better repeat with all available methods.  Maybe, with an include statement as a snippet (we can do this later).

For the moment, we could add here at the beginning of the note, as the first sentence:

> "Keep in mind that before actually installing (deploying) the proxy you must probably adjust the Squid cache size."